scripts/affected-checks.ps1

```
@@ -0,0 +1,125 @@
+        git diff --name-only origin/main...HEAD
+    }
+}
+
+Write-Host "🔎 Determinando archivos cambiados..." -ForegroundColor Cyan
+$changed = Get-ChangedFiles -Staged:$Staged -Ci:$Ci | Where-Object { $_ -ne '' }
+if (-not $changed) { Write-Host "No changed files detected."; exit 0 }
+$changed | ForEach-Object { Write-Host "  • $_" }
+
+# Filter relevant files
+$relevant = $changed | Where-Object { $_ -match '\.rs$' -or $_ -match 'Cargo\.toml$' -or $_ -match 'build\.rs$' }
+if (-not $relevant) { Write-Host "No relevant Rust files changed."); exit 0 }
```

PSScriptAnalyzer (1.23.0)

[error] Missing closing '}' in statement block or type definition.

(MissingEndCurlyBrace)